testnets/bfhevmtestnet/assetlist.json

```diff
@@ -0,0 +1,38 @@
+{
+    "$schema": "../assetlist.schema.json",
```

Comment:
Sorry, another minor detail, the relative path to the schema must also be updated (add another ../) (i.e., `"$schema": "../../assetlist.schema.json",`). The same also applied to versions and chain jsons